packages/roosterjs-content-model-core/lib/coreApi/formatContentModel/formatContentModel.ts

```diff
@@ -81,7 +81,7 @@ export const formatContentModel: FormatContentModel = (
                 contentModel: clearModelCache ? undefined : model,
                 selection: clearModelCache ? undefined : selection,
                 source: options?.changeSource || ChangeSource.Format,
-                data: getChangeData?.(),
+                data: options?.getChangeData?.() || getChangeData?.(),
```

Comment:
hmm I don't understand, getChangedData() is from options, so options.getChangedData should be always same with getChangedData.